modules/apps/commerce/commerce-payment-method-money-order/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -25,11 +25,9 @@ String messageAsLocalizedXML = moneyOrderGroupServiceConfiguration.messageAsLoca
 
 		<div id="<portlet:namespace />message">
 			<aui:field-wrapper label="message">
-				<liferay-ui:input-localized
-					fieldPrefix="settings"
-					fieldPrefixSeparator="--"
+				<liferay-editor:input-localized
+					defaultLanguageId="<%= themeDisplay.getLanguageId() %>"
 					name="messageAsLocalizedXML"
-					type="editor"
 					xml="<%= messageAsLocalizedXML %>"
```

Comment:
To make up for the lack of fieldPrefix and fieldPrefixSeparator, we can change the way we are retrieving and saving those fields on the backend(EditMoneyOrderCommercePaymentMethodConfigurationMVCActionCommand).